src/ui/components/Library/Team/View/NewTestRuns/suspense/TestRunsCache.ts

```diff
@@ -0,0 +1,32 @@
+import { createIntervalCache } from "suspense";
+
+import { GraphQLClientInterface } from "shared/graphql/GraphQLClient";
+import { TestRun, processTestRun } from "shared/test-suites/TestRun";
+import { getTestRunsGraphQL } from "ui/components/Library/Team/View/TestRuns/graphql/TestRunsGraphQL";
+
+export const testRunsIntervalCache = createIntervalCache<
+  number,
+  [graphQLClient: GraphQLClientInterface, accessToken: string | null, workspaceId: string],
+  TestRun
+>({
+  debugLabel: "testRunsIntervalCache",
+  getPointForValue: testRun => new Date(testRun.date).getTime(),
```

Comment:
@bvaughn Is there a way to reverse the order with interval cache? Unless I'm doing something silly 😅 I think I can reverse it by using negative values (-getPointForValue, -start and -end) and them swap with each other, then call the api with start:+end, end:+start. But it feel really weird and unintuitive.

Reply:
Pushed up changes to show you what I mean. https://github.com/replayio/devtools/pull/9901/commits/911f03b520f185dd582996643690614181adcbfa

Reply:
No, caches don't support reverse ordering. Seems like that would be unnecessary complexity? If you want to display cached values in reverse, just iterate over the collection backward?

Reply:
> memoized .sort()

Yes! This works great, thanks!